third_party/move/move-model/src/model.rs

```diff
@@ -65,13 +68,16 @@ use move_core_types::{
 };
 use move_disassembler::disassembler::{Disassembler, DisassemblerOptions};
 use num::ToPrimitive;
+use once_cell::sync::Lazy;
 use serde::{Deserialize, Serialize};
 use std::{
     any::{Any, TypeId},
+    backtrace::{Backtrace, BacktraceStatus},
     cell::{Ref, RefCell, RefMut},
     collections::{BTreeMap, BTreeSet, VecDeque},
     ffi::OsStr,
     fmt::{self, Formatter, Write},
+    ops::Deref,
```

Comment:
There is no need to ever import this and use `deref` explicitly

Reply:
We are able to avoid it in this case, anyway.

Fixed.